frontend/src/index.js

```diff
@@ -0,0 +1,55 @@
+let loggedInAccount
```

Comment:
need ';' throughout

Reply:
I don't think we need them (they are optional in most cases in javascript), although if we decide that's what we want to do I can definitely add them